### ISSUE TITLE
Why Test result has red spot?

### ISSUE THREAD

**Issue by ymzlygw** [MAlberts99/PyTorch-AdaIN-StyleTransfer#2]
Hi, First thanks for your Implement for AdaIn.
I want to know why I get the following result when I use the 'Style Transfer Test.ipynb'. 
I used the pretrained mode of vgg and decoder as read.md.

![ダウンロード](https://user-images.githubusercontent.com/17065855/108966432-8100a180-76c1-11eb-8baf-2d8dcdda60ea.png)

As you can see, the result image has many red spots and I don't know why, can you explain about it? thanks
By the way,I delete following part in processing of network output:
"out = torchvision.utils.make_grid(out.clamp(min=-1, max=1), nrow=3, scale_each=True, normalize=True)"
And Use:
img = toPIL(**out[0]**)
to get the red spotted image
And these red parts is the black color in the origin content image. is there some relation?

**Comment by WendongZh** [comment]
In my opinion, the reason for these red spots may come from that some pixels contain extreme values, such as exactly 1 or  -1.

So you can try to remove the 'clamp' operation in the code, like:

"out = torchvision.utils.make_grid(out, nrow=3, scale_each=True, normalize=True)"

This may influence the performance of the model to some extend, but it can avoid the red spots in my case.

Besides, I also suggest you reading the original code at https://github.com/xunhuang1995/AdaIN-style
